domain-management/src/main/java/org/jboss/as/domain/management/security/FileKeyManagerService.java

```diff
@@ -21,26 +21,49 @@
  */
 package org.jboss.as.domain.management.security;
 
-import java.security.KeyStore;
-
 import org.jboss.as.controller.services.path.PathEntry;
 import org.jboss.as.controller.services.path.PathManager;
 import org.jboss.as.controller.services.path.PathManager.Callback;
 import org.jboss.as.controller.services.path.PathManager.Event;
 import org.jboss.as.controller.services.path.PathManager.PathEventContext;
+import org.jboss.as.domain.management.logging.DomainManagementLogger;
 import org.jboss.msc.inject.Injector;
-import org.jboss.msc.service.StartContext;
 import org.jboss.msc.service.StartException;
 import org.jboss.msc.service.StopContext;
 import org.jboss.msc.value.InjectedValue;
+import sun.security.x509.AlgorithmId;
```

Comment:
The problem we are going to have here is the IBM JDK, although to start with we could detect if running on the IBM JDK and make this feature unavailable.